build/int.cloudbuild.yaml

```
@@ -64,6 +64,6 @@ tags:
 - 'integration'
 substitutions:
   _DOCKER_IMAGE_DEVELOPER_TOOLS: 'cft/developer-tools'
-  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1.4.13'
+  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1.20.7'
```

Whoops, `1.20.7` is not a valid version today. I'll fix my commit.
```suggestion
  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1.14.4'
```